frontend/src/core/comments/components/CommentsList.css

```diff
@@ -24,3 +24,8 @@
     flex-flow: column wrap;
     width: 100%;
 }
+
+.pinned-comments {
+    border-bottom: 1px solid #5e6475;
```

Comment:
Let's follow the styling of Preferred Locales in the Locales tab and use `#7bc876` as the border color.

Same as in the Locales panel, I'd also use the full-width the border line, by moving padding from the parent element to `ul` and `div` elements.